app/scripts/MoveableTrack.js

```diff
@@ -4,15 +4,14 @@ import React from 'react';
 import DraggableDiv from './DraggableDiv';
 import TrackArea from './TrackArea';
 
-import { isWithin } from './utils';
-
-
-const checkMousePosVsEl = (x, y, el) => {
-  const bBox = el.getBoundingClientRect();
-  return isWithin(
-    x, y, bBox.left, bBox.left + bBox.width, bBox.top, bBox.top + bBox.height
-  );
-};
+// See commented out block in "render()".
```

Comment:
Was this function not being used or what?

Reply:
No: It's referenced in this commented out block in `render`, but that's the only usage. Since the block in `render` was kept around, rather than just deleted, I treated this function the same way.
```
        onMouseLeave={(/* e */) => {
          // This let to https://github.com/higlass/higlass/issues/263
          // Therefore I disabled it.
          // if (checkMousePosVsEl(
          //   e.nativeEvent.clientX, e.nativeEvent.clientY, this.el
          // )) {
          //   return;
          // }
          this.handleMouseLeave();
        }}
```